src/components/chips/demoValidation/script.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  'use strict';
+  angular
+    .module('chipsValidationDemo', ['ngMaterial', 'ngMessages'])
+    .controller('ChipsValidationCtrl', ValidationCtrl);
+
+  function ValidationCtrl () {
+    this.selectedFruit = [];
+    this.selectedVegetables = [];
+  }
+})();
```

Comment:
Blank line at end of file please.

Reply:
Is this fixed @Splaktar ? Because in the code it seems like, but just the one comment of you got hidden, this here not.

Reply:
Yep, looks good now.